mini_Tutorial.md

```diff
@@ -179,7 +179,7 @@ will summarize all the information for us
 
 ```
 $ python annotate_tree.py -t 10239 \
--db /path/to/ete3_db \
+-db /path/to/ete3_db/ \
```

Comment:
No, it shouldn't make a difference but I did run into an error because I thought it was a file instead of a directory. ete3_db is not mentioned before, so hence my confusion.